mkdocs.yml

```diff
@@ -182,6 +183,7 @@ nav:
         - Consensus protocols:
           - Overview: Concepts/Consensus-Protocols/Overview-Consensus.md
           - Comparing PoA consensus protocols: Concepts/Consensus-Protocols/Comparing-PoA.md
+        - Data storage formats: Concepts/Data-Storage-Formats.md
```

Comment:
I would put this lower on the Concepts list, maybe under Monitoring? I don't think we have a set rule for ordering navigation topics; it can get hairy when there are so many sub-topics. Perhaps this is something to discuss in order to make the docs more consistent and readable.

Reply:
It looks like they started off in alphabetical order, up until Monitoring, that's why I originally put it there.  I think it makes sense to reorder them all that way, what do you think?